### DISCUSSION TITLE
Branch cleaning, last couple of branches left, info needed

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3277] [Questions & Answers] [answered]
Hi everybody,
we've finally finished to rebase of all of PRs after the approval of https://github.com/OpenMage/rfcs/blob/main/accepted/0002-release-schedule.md, this allowed us to do some housekeeping cleaning to our branches.

We still have a couple of branches that are not directly related to software versions (https://github.com/OpenMage/magento-lts/tree/replace-1525 for https://github.com/OpenMage/magento-lts/pull/2940 and https://github.com/OpenMage/magento-lts/tree/functional-test-suite), these branches should remain alive until those features are merged or canceled.

But this post is about 2 other branches:
- https://github.com/OpenMage/magento-lts/tree/sreichel-check-frontend-files could have been created by mistake? because https://github.com/OpenMage/magento-lts/pull/2941 is linked to another branch in another repo
- https://github.com/OpenMage/magento-lts/tree/feature/admin_captcha_settings doesn't seem to be related to any PR, is it empty, is it used? I don't know.

last activities on those branches are by @sreichel, do you remember if you're using or you have some important data on those branches? or could we remove them?

thanks everybody, sorry for the boring post :-D

**Comment by sreichel** [comment, marked as answer]
Deleted both.

**Comment by addison74** [comment]
Many good things have happened and are visible. Now we would need an effort from those who have been active for a while to solve the remaining PRs and reported issues, which are not as many as 1 year ago, so that later we can focus our attention through the Discussions-Ideas and Don't Forget PRs.
